draft-barnes-acme.md

```diff
@@ -1459,14 +1450,131 @@ TODO
+
+In this section, we discuss the threat model that underlies ACME and the ways that ACME achieves these security goals within that threat model.  We also discuss the denial-of-service risks that ACME servers face, and a few other miscellaneous considerations.
+
+## Threat model
+
+As a service on the Internet, ACME broadly exists within the Internet threat model {{RFC3552}}.  In analyzing ACME, it is useful to think of an ACME server interacting with other Internet hosts along three "channels":
+
+* An ACME channel, over which the ACME HTTPS requests are exchanged
+* A validation channel, over which the ACME server performs additional requests to validate a client's control of an identifier
+* A contact channel, over which the ACME server sends messages to the registered contacts for ACME clients
+
+~~~~~~~~~~
++------------+
+|    ACME    |     ACME Channel
+|   Client   |--------------------+
++------------+                    |
+       ^                          V
+       |   Contact Channel  +------------+
+       +--------------------|    ACME    |
+                            |   Server   |
+                            +------------+
++------------+                    |
+| Validation |<-------------------+
+|   Server   |  Validation Channel
++------------+
+~~~~~~~~~~
+
+In practice, the risks to these channels are not entirely separate, but they are different in most cases.  For example, a CDN employed by the ACME server might be able to tampler with a ACME messages, but not read the client's email (contact channel) or change DNS records for the client's domain names (validation channel).  
+
+Broadly speaking, ACME aims to be secure against active and passive attackers on any individual channel.  Some vulnerabilities arise (noted below), when an attacker can exploit both the ACME channel and one of the others.
+
+On the ACME channel, in addition to network-layer attackers, we also try to defeat an application-layer man in the middle.  Having protections for this case allows an ACME-enabled CA to make use of intermediaries such as Content Distribution Networks (CDNs) with minimal risk.  It also provides defense in depth, e.g., to address cases where a client is behind a middlebox that acts as a man in the middle.
+
+## Integrity of Authorizations
+
+ACME allows anyone to request challenges for an identifier, by registering an account key and sending a new-authorization request under that account key.  The integrity of the authorization process thus depends on the identifier validation challenges to ensure that the challenge can only be completed by someone who both (1) holds the private key of the account key pair, and (2) controls the identifier in question.
+
+All of the challenges above that require an out-of-band query by the server have a binding to the account private key, such that the only the account private key holder can successfully respond to the validation query:
+
```

Comment:
This property is *not* provided by unique tokens.  If that were the only assurance, then an ACME MitM could get the real domain owner to provision a validation response that the server believes to be from the MitM's private key.  That's why you need the binding to the private key.  I'll try to clarify.